providers/fips/fipsprov.c

```diff
@@ -695,6 +695,8 @@ int OSSL_provider_init_int(const OSSL_CORE_HANDLE *handle,
         }
     }
 
+    OPENSSL_cpuid_setup();
```

Comment:
This is too late.  The default entry point (DEP) runs the self tests and it is called when the shared object is loaded.  This happens too late.

This is a FIPS mandate unfortunately.

Reply:
Oops, forgot we took that approach....
Ignore me :)